### PR TITLE
Stub

### PR DESCRIPTION
**Pull request by maximusunc**
The service still doesn't do anything, but this stub is to help the ARS get it incorporated in with their service.

### CODE REVIEW

File: app/ordering_components.py [hunk 1]
```diff
@@ -1,37 +1,33 @@
...
-async def get_confidence(result, message):
+"""Compute scores for each result in the given message."""
+
+
+def get_confidence(result, message, logger):
     # TODO Implement actual g()-score
     score_sum = 0
     score_count = 0
-    for analysis in result:
+    for analysis in result["analyses"]:
```
Comment (kennethmorton) [inline app/ordering_components.py]: use get() for message parsing saftey?

File: run.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+if __name__ == "__main__":
+    uvicorn.run(
+        "app.server:APP",
+        host="0.0.0.0",
+        port=9096,
+        reload=True,
+        reload_dirs=["app"],
+        log_config=None,
+    )
```
Comment (kennethmorton) [inline run.py:12]: Perhaps we should snag some these inputs from a CLI and specify workers?

**Review by kennethmorton — APPROVED** [review]
Looks good. Minor comments.

File: app/ordering_components.py (hunk 1 above)
Comment (maximusunc) [reply app/ordering_components.py]: Sure. Analyses are required inside each result, but I agree it's good form to be sure.

File: run.py (hunk 2 above)
Comment (maximusunc) [reply run.py:12]: I'm not sure this is necessary. When the ARS goes to deploy, they'll be using helm charts where the port will be specified and number of workers. This run.py file was just for my development (with automatic reloads on code change), and when the docker image is used in deployment, the main.sh file is the one that's uses which allows a port to be specified.

**Pull request merged**